netdata.spec.in

```diff
@@ -185,9 +185,15 @@ Requires(pre): /usr/sbin/useradd
 # #####################################################################
 # Functionality-dependent package dependencies
 # #####################################################################
-# Note: Some or all of the Packages may be found in the EPEL repo, 
+# Note: Some or all of the Packages may be found in the EPEL repo,
 # rather than the standard ones
 
+# epbf dependencies
+%if 0%{?_have_ebpf}
+BuildRequires:  elfutils-libelf-devel
+%endif
```

Comment:
```suggestion
%if 0%{?_have_ebpf}
%if 0%{?suse_version}
BuildRequires: libelf-devel
%else
BuildRequires:  elfutils-libelf-devel
%endif
%endif
```

This change is just to make things work correctly on SUSE (who insist on having significantly different package names from most other RPM distros).

Reply:
Find it on `centos-stream-8-x86_64` config firstly. Added your changes